tools/spectral/.spectral.yaml

```
@@ -122,6 +122,15 @@ rules:
       functionOptions:
         match: "^[A-Za-z0-9-._~:/?#\\[\\]@!\\$&'()*+,;=]*$"
 
+  xgen-docs-operationId-camel-case:
+    description: operationId should be camelCased.
+    given: "$.paths[*]..operationId"
+    severity: error
+    then:
```

can you add a message like 
```suggestion
    message: "Operation ID should be camelCase: {{property}}."
    then:
```
so we can see what the current value is?

---

good point! I will update it

---

I updated to use `{{value}}` which gives the message "40221:26    error  xgen-docs-operationId-camel-case  Operation ID should be camelCase: grantMongoDBEmployeeAccess."

Using `{{property}}`` does not give a useful message  "40221:26    error  xgen-docs-operationId-camel-case  Operation ID should be camelCase: operationId."

---

perfect I wasn't sure what the right interpolation was, thanks for fixing it